operator/tests/common/delete-securedcluster-errors.yaml

```diff
@@ -293,7 +293,7 @@ metadata:
 apiVersion: authorization.openshift.io/v1
 kind: RoleBinding
 metadata:
-  name: sensor-use-scc
+  name: sensor-use-sccs-for-sensor
```

Comment:
I appreciate the clear naming, but one thing to keep in mind is that in the manual upgrade instructions, you will need to specify `oc rm scc ... ; oc create -f - <<EOF ...` commands for all the renames.

Reply:
Ok, this convinces me to stay with old names :)